src/js/components/aboutTheData/AboutTheDataPage.jsx

```diff
@@ -105,13 +112,18 @@ const AboutTheDataPage = ({
         }
     };
 
+    console.log('search', search);
+
     return (
         <div className="about-the-data about-the-data_agencies-page">
             <Header />
             <StickyHeader>
                 <div className="sticky-header__title">
                     <h1 tabIndex={-1}>Agency Submission Statistics</h1>
                 </div>
+                <div className="sticky-header__toolbar">
+                    <ShareIcon slug={`submission-statistics/${search}`} email={emailDetails} />
```

Comment:
It looks like the linkedIn link goes to the homepage. And so does the Covid link.

Reply:
There are tests for this. It may be an issue with linkedin that changed or something or may have always been like that.